servers/src/main/java/tachyon/master/file/options/SetAclOptions.java

```diff
@@ -112,10 +112,10 @@ public Builder setOperationTimeMs(long operationTimeMs) {
     }
 
     /**
-     * Builds a new instance of {@link SetAclOptions}.
+     * Builds a new instance of {@link SetAclOptions}. Throws an {@link IllegalArgumentException} if
```

Comment:
I think in the rest of this codebase, we didn't really document this exception which is runtime exception.
So I think we should probably not document it.